### PR TITLE
adding left

### PR DESCRIPTION
**Pull request by wjlancas**
Mic drop

### CODE REVIEW

File: left.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+left
```
Comment (rifaiwael) [inline left.txt:1]: No way man, are you dropping the mic that early?

Comment (wjlancas) [inline left.txt:1]: wtf

Comment (rifaiwael) [inline left.txt:1]: you have to listen, I am the owner!!!

Comment (wjlancas) [inline left.txt:1]: look at my file @rifaiwael

**Pull request merged**